lib/asn1.js

```diff
@@ -916,6 +1062,9 @@ asn1.validate = function(obj, v, capture, errors) {
         if(v.captureAsn1) {
           capture[v.captureAsn1] = obj;
         }
+        if(v.captureRaw && 'raw' in obj) {
```

Comment:
I'm wondering if `captureDer` would be a better name. It wouldn't work map cleanly with BER-formatted data, unfortunately.